code/modules/1713/weapons/guns/rocket.dm

```diff
@@ -83,14 +83,15 @@
 	load_delay = 18
 
 /obj/item/weapon/gun/launcher/rocket/examine(mob/user)
-	..()
-	if (max_rockets > 1)
-		user << SPAN_NOTICE("<b>LOADED [rockets.len]/[max_rockets]</B>")
-	else
-		if (rockets.len)
-			user << SPAN_NOTICE("<b>LOADED</B>")
-		else
-			user << SPAN_NOTICE("<b>UNLOADED</B>")
+    ..()
+    if (!istype(src, /obj/item/weapon/gun/launcher/rocket/single_shot)) // Check if not single_shot or its subtypes
```

Comment:
You could've added an else statement here instead of using `/obj/item/weapon/gun/launcher/rocket/single_shot/examine(mob/user)`

Reply:
its better to have seperate examine procs though i thought for readability